### PR TITLE
Data modified by editing JSON files. Names on website updated.

### PR DESCRIPTION
**Pull request by psurya1994**

### CODE REVIEW

**Comment by sudheesh001** [comment]
@psurya1994 Why has some people's name been deleted ?

**Comment by psurya1994** [comment]
@sudheesh001 Do you think we should have the names of students who have contributed less with photo on the home page? I thought it'd be better if we have photos only for the main contributors and we'll have the names of other contributors in one line (without the photos). 

I replaced the JSON file with names of main contributors and typed names of others manually. Let me know what you think.

File: index.html [hunk 1]
```diff
@@ -188,13 +188,14 @@ <h4 class="service-heading">Where Are We Now?</h4>
...
                 <div class="col-lg-12 text-center">
-                    <h1>Contributors</h1>
+                    <h1>The Team</h1>
                     <br>
                 </div>
             </div>
             <div class="row" id="contrib">
                 <!-- contributors added here dynamically -->
             </div>
+            <h4>Other Contributors:</h4>Aarti Barai, Sriram Kovela, Bharath Kumar, Vinay Kant, Aman Bhargava, Nisha Jacob, Satya Kesav, Anirudh Deshpande, Milind Sheth, Sreetam Das, Milan Chatterjee, Chetan Rathore, Surendra Gurjar, Shubhi Saxena, Devendra Patil, Shivani Shukla, Spandan Pandey, Aroma Rodrigues.</h4>
```
Comment (psurya1994) [inline index.html]: @sudheesh001 Names of minor contributors updated here.

Comment (sudheesh001) [inline index.html]: What's the metric for the major contribution ?

Comment (psurya1994) [inline index.html]: Hmm, didn't think about where exactly we'll draw the line. I considered people who contributed more than one project, and have continued to work as major contributors.

But I think we have about 30 contributors, and it doesn't make sense to have photos of all of them. Or does it?

**Comment by psurya1994** [comment]
@sudheesh001 Do you think we should have even the minor contributors with photos on the site? I'm not really sure how this generally works.

**Comment by psurya1994** [comment]
@sudheesh001 any updates?

**Comment by sudheesh001** [comment]
As discussed, Maybe we could have a separate page /contributors.html which contains the list and images of all the contributors. As of now this patch looks good, but those changes and this will be good to merge.

**Pull request merged**